문자열-다루기-기본.js

```diff
@@ -0,0 +1,7 @@
+function solution(s) {
+    let answer = false;
+    if (s.length === 4 || s.length === 6) {
+        answer = s.split("").every((v) => !isNaN(v));
+    }
+    return answer;
```

Comment:
```suggestion
    if (s.length === 4 || s.length === 6) {
        return s.split("").every((v) => !isNaN(v));
    }
    return false;
```
every 메소드가 불리안 타입 자체를 리턴해서 저렇게도 표현 가능할거같아요. 뭐가 더 좋은 코드인지는 개발자의 성향이 따라 다르다고 생각하는데 저렇게 표현하는 방법도 있다는 걸 알아두시면 좋을거같습니다.